zuul.d/doc-jobs.yaml

```diff
@@ -1,4 +1,18 @@
 ---
+- job:
+    name: build-otc-doc
+    description: |
+      Build document with Sphinx without tox.
+    timeout: 1800
+    vars:
+      sphinx_build_dir: doc/build
+      sphinx_source_dir: doc/source
+      sphinx_pdf_files:
+        - "doc-{{ zuul.project.short_name }}.pdf"
+    pre-run: playbooks/docs/pre.yaml
+    run: playbooks/docs/run.yaml
+    post-run: playbooks/docs/post.yaml
```

Comment:
is enough fedora pod here? maybe `ubuntu-jammy`?

Reply:
neah, this should just work anywhere. We will request particular node-set (not default) only when explicitly necessary (i.e. too many files in git to run in pod)